src/components/MailingList/MailingList.test.jsx

```diff
@@ -0,0 +1,19 @@
+import axios from "axios";
+
+// NOTE: this test does NOT consitute a thorough test of the MailingList.
+// It simply ensures that the proper environment variables are installed.
+it("can issue a non-404-ing request to the backend", () => {
+    let mh_backend = process.env.REACT_APP_MH_BACKEND_URL;
+    let payload = {};
+    // An empty payload is a confirmed noop on the Mailjet side.
+    return axios({
```

Comment:
was going to ask. do we really _need_ `axios`? thoughts on refactoring to the Fetch API?

Reply:
We don't need `axios`, and I'd definitely prefer to refactor.